src/frontends/pytorch/src/utils.hpp

```diff
@@ -20,6 +20,17 @@ class FrameworkNode;
 namespace frontend {
 namespace pytorch {
 
+/// \brief Macro to check whether a boolean condition holds.
+/// \param COND Condition to check
+/// \param ... Additional error message info to be added to the error message via the `<<`
+///            stream-insertion operator. Note that the expressions here will be evaluated lazily,
+///            i.e., only if the `cond` evalutes to `false`.
+/// \throws ::ov::frontend::OpConversionFailure if `cond` is false.
+#ifndef PYTORCH_OP_CONVERSION_CHECK
+#    define PYTORCH_OP_CONVERSION_CHECK(COND, ...) \
+        OPENVINO_ASSERT_HELPER(::ov::frontend::OpConversionFailure, "", (COND), "[PyTorch Frontend] " __VA_ARGS__)
```

Comment:
Adding this prefix can increase binary size as on every usage of this macro the additional literal text will be added.

It requires some experiments to check bin-size but the 2nd parameter is context and the additional text can be put there  e.g as some function which return const reference to global pytorch FE prefix string. Then when exception is created the reference will be used (instead literal) to build exception message.

Reply:
> some function which return const reference to global pytorch FE prefix string

Done.

>  the 2nd parameter is context and the additional text can be put there

I didn't use 2nd parameter here, as context is not added to exception description, but to the previous line. In this case exception text look like this:

```
E       FrontEnd API failed with OpConversionFailure[PyTorch Frontend] :
E       Not expected number of inputs for aten::arange
```

I need the exception text look like this:

```
E       FrontEnd API failed with OpConversionFailure:
E       [PyTorch Frontend] Not expected number of inputs for aten::arange
```

> It requires some experiments to check bin-size:

Size of bin folder on this branch:
510,1 MiB (534.901.650 bytes)

Size of bin folder on master without my changes:
509,5 MiB (534.203.474 bytes)

Reply:
700K bytes is significant for such small changes. Agree with @praasz, and let us fix it in the next PR.

Reply:
I think would be better to make as separate task.
Also consider to propose change that others FE could use it to give own prefix for better exception message identification.

Regarding error message format I expected is little different then current implementation but it looks like this context is not used right now. So if common exception `create`, `make_what` will be updated correct it could give same error message format but also gives other FE to put custom text if required.

Reply:
I think we should create exception class in OV that will report component name automatically where exception is raised. Also, it needs functionality that will split sensitive and unsensitive parts of the message so we can send non-sensitive part to telemetry and report both parts to users locally.